ntoskrnl/io/pnpmgr/devaction.c

```diff
@@ -248,12 +248,12 @@ IopCreateDeviceInstancePath(
 
     DPRINT("Sending IRP_MN_QUERY_CAPABILITIES to device stack (after enumeration)\n");
 
-    Status = IopQueryDeviceCapabilities(DeviceNode, &DeviceCapabilities);
+    Status = PiIrpQueryPnPDeviceCapabilities(DeviceNode, &DeviceCapabilities);
```

Comment:
Hm, let's just pass device capabilities into `IopCreateDeviceInstancePath` as a parameter, so `PiInitializeDevNode` makes only one `IRP_MN_QUERY_CAPABILITIES` IRP

Reply:
Ok, i'll have a look on that.